### PR TITLE
Bindu | BAH-1468 | Remove deprecated spring annotation handler mappings

### PR DESCRIPTION
**Pull request by binduak**
https://bahmni.atlassian.net/browse/BAH-1468

### CODE REVIEW

File: omod/src/main/resources/webModuleApplicationContext.xml [hunk 1]
```diff
@@ -20,7 +20,5 @@
   		    http://www.springframework.org/schema/util
   		    http://www.springframework.org/schema/util/spring-util-3.0.xsd">
 
-	<bean class="org.springframework.web.servlet.mvc.annotation.DefaultAnnotationHandlerMapping"/>
-	
-	<context:component-scan base-package="org.openmrs.module.episodes.web.controller" />
+	<context:component-scan base-package="org.openmrs.module.episodes" />
```
Comment (angshu) [inline omod/src/main/resources/webModuleApplicationContext.xml]: why not make it narrower? "org.openmrs.module.episodes.web"

Comment (binduak) [reply omod/src/main/resources/webModuleApplicationContext.xml]: @angshu There is no "web" package in the omod

Comment (angshu) [reply omod/src/main/resources/webModuleApplicationContext.xml]: then the component scan should be in the API module moduleApplicationContext.xml, no? also, I would have preferred to add a ".api" package and pushing all daos, services under that. I am ok right now, if you move this to the API applicationcontext

Comment (binduak) [reply omod/src/main/resources/webModuleApplicationContext.xml]: @angshu Removed unnecessary component-scan

**Review by angshu — APPROVED** [review]

File: omod/src/main/resources/webModuleApplicationContext.xml (hunk 1 above)
Comment (binduak) [reply omod/src/main/resources/webModuleApplicationContext.xml]: @angshu Created the tech debt card [https://bahmni.atlassian.net/browse/BAH-1891](https://bahmni.atlassian.net/browse/BAH-1891)

**Pull request merged**